agendas/2022/2022-05-05.md

```diff
@@ -113,3 +114,14 @@ This is an open meeting in which anyone in the GraphQL community may attend.
    - [Ready for review](https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Ready+for+review+%F0%9F%99%8C%22+sort%3Aupdated-desc)
    - [All open action items (by last update)](https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Action+item+%3Aclapper%3A%22+sort%3Aupdated-desc)
    - [All open action items (by meeting)](https://github.com/graphql/graphql-wg/projects?query=is%3Aopen+sort%3Aname-asc)
+1. Discuss new intersection type proposal (15m, Yaacov)
+   - Alternative to unions implementing interfaces discussed at last WG
+   - spec PR: https://github.com/graphql/graphql-spec/pull/941
+   - graphql-js PR: https://github.com/graphql/graphql-js/pull/3550
+   - discussion: https://github.com/graphql/graphql-wg/discussions/944
+1. Defer/Stream
+   - Asynchronous iterators of iterables versus of items, continuation of last WG meeting (15 min, Yaacov)
+     - Should resolvers returning AsyncIterables returning AsyncIterables of items or AsyncIterables of Iterables of items? 
+     - Should the AsyncGenerator returned by graphql-js yield items singly or yield all available items (in array or otherwise)? 
+     - Background: https://github.com/robrichard/defer-stream-wg/discussions/38
+     - See above link for polls, still waiting for feedback explaining votes.
```

Comment:
```suggestion
1. Defer/Stream: asynchronous iterators of iterables versus of items, continuation of last WG meeting (15 min, Yaacov)
   - Should resolvers returning AsyncIterables returning AsyncIterables of items or AsyncIterables of Iterables of items? 
   - Should the AsyncGenerator returned by graphql-js yield items singly or yield all available items (in array or otherwise)? 
   - Background: https://github.com/robrichard/defer-stream-wg/discussions/38
   - See above link for polls, still waiting for feedback explaining votes.
```